packages/core/src/core/prompts.ts

```diff
@@ -153,7 +158,25 @@ ${(function () {
   }
   return '';
 })()}
-
+${(function () {
```

Comment:
Anything that messes with the core system prompt makes me nervous. Think this is good. happy to approve and get rolling, lets be open to revisiting if need be based on any follow on feedback?